extensions/amp-iframe/1.0/amp-iframe.js

```diff
@@ -31,6 +32,36 @@ class AmpIframe extends BaseElement {
     );
     return super.isLayoutSupported(layout);
   }
+
+  /** @override */
+  init() {
+    return dict({
+      'onLoadCallback': () => {
+        const placeholder = this.getPlaceholder();
+        if (placeholder) {
+          this.togglePlaceholder(false);
+        } else {
+          const pos = this.element./*OK*/ getLayoutBox();
+          const minTop = Math.min(
+            600,
+            this.getViewport().getSize().height * 0.75
```

Comment:
Ditto, can this value be read from the `IntersectionEntry`?

Reply:
Viewport height cannot be read from `IntersectionEntry`

Reply:
Discussed offline: It can, it's `rootBounds`. Thanks for the change.